Dockerfile

```diff
@@ -31,6 +31,10 @@ ARG IRONIC_SOURCE
 ARG IRONIC_INSPECTOR_SOURCE
 ARG SUSHY_SOURCE
 
+COPY sources /sources/
```

Comment:
I wonder if this can be an ARG?

Reply:
I'd rather keep all the sources under a standard path
it needs to exist (otherwise the copy fails) and it needs to be under the same context as the Dockerfile, so I don't see a lot of margin for customization

Reply:
Same context, right.. This is unfortunate, but not too bad.